examples/proc_macro_commands.rs

```diff
@@ -0,0 +1,67 @@
+use redis_module::{redis_module, Context, RedisResult, RedisString, RedisValue};
+use redis_module_macros::redis_command;
+
+#[redis_command(
+    {
+        name: "classic_keys",
+        flags: "readonly",
+        arity: -2,
```

Comment:
So again, I believe this is a future work when we will process the function arguments. It will not be a breaking change because we will change the `arity` to be optional.

Reply:
Yes sure, I just do not want to do it know because we just release 2.0